### ISSUE TITLE
Error while installing using npm 

### ISSUE THREAD

**Issue by mohammedzamakhan** [rackerlabs/snappit-mocha-protractor#29]
I am getting the following error on both Windows and Mac machines

```
npm ERR! canvas@1.2.3 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the canvas@1.2.3 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the canvas package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls canvas
npm ERR! There is likely additional logging output above.
```

**Comment by Droogans** [comment]
@mohammedzamakhan Hi! I apologize for not responding to this issue for so long. I had disabled all watching on my repositories for work and did not enable it for this one!

I am very sorry to leave you hanging like this for a couple of months....are you still having this issue? Did you resolve it?

I believe I can address this by highlighting install instructions for mac/linux for getting the required image manipulation libraries installed [that `node-canvas` needs](https://github.com/Automattic/node-canvas#installation).

**Comment by Droogans** [comment]
I have included a helper script in version 0.2.0 that will attempt to find and install all system-level image manipulation dependencies for mac and (some distros of) linux.

Unfortunately, Windows requires that visual studio is installed and configured, which is outside the scope of this project.

I'll write some documentation and link to it here. For now, upgrade to 0.2.0, and run

```
./node_modules/.bin/snappit-install
```

and let me know if that helps.

**Comment by Droogans** [comment]
Update: I have included some steps [in the wiki](https://github.com/rackerlabs/snappit-mocha-protractor/wiki#introduction) for grabbing system-level image manipulation dependencies.

Please let me know if that addresses your issue.
